docker-compose.yml

```diff
@@ -1,15 +1,51 @@
 version: '3.8'
 
 services:
+  
+  postgres:
+    image: postgres:17
+    container_name: postgres-db
+    restart: unless-stopped
+    environment:
+      POSTGRES_USER: myuser
+      POSTGRES_PASSWORD: mypassword
+      POSTGRES_DB: mydatabase
+    volumes:
+      - pgdata:/var/lib/postgresql/data
+      - ./backend/db/schema.sql:/docker-entrypoint-initdb.d/schema.sql
+    networks:
+      - dev-net
+    ports:
+      - "5432:5432"
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U myuser -d mydatabase"]
+      interval: 5s
+      timeout: 5s
+      retries: 5
+  
   devcontainer:
     build:
       context: .
       dockerfile: Dockerfile
```

Comment:
Please either change the file path of the Docker file here to be `dockerfile` since that is currently how the file is named or rename it to match the case in the `docker-compose.yml`.